design-documents/web-api-controllers.md

```diff
@@ -0,0 +1,374 @@
+# Controllers for web APIs, user input processing on top of service contracts execution
```

Comment:
I would propose to remove "Controllers" and add something like middleware.

Reply:
Agreed